srcpkgs/ntl/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'ntl'
+pkgname=ntl
+version=11.3.2
+revision=1
+wrksrc=$pkgname-$version/src
+build_style=configure
+configure_args="DEF_PREFIX=/usr"
```

Comment:
you will add a lot to this due to the build system being homemade horrible.

please add the following, cut at 80 columns.

```
PREFIX=${XBPS_CROSS_BASE}/usr CC=${CC} AR=${AR} RANLIB=${RANLIB} SHARED=ON NATIVE=OFF
```

Reply:
If CC or CXX is set, build will fail